interfaces/builtin/network_control.go

```diff
@@ -119,6 +119,7 @@ network sna,
 /{,usr/}{,s}bin/routel ixr,
 /{,usr/}{,s}bin/rtacct ixr,
 /{,usr/}{,s}bin/rtmon ixr,
+/{,usr/}{,s}bin/ss ixr,
```

Comment:
This is fine since it only allows profile inheritance to happen (no new permissions)